colour/models/rgb/tests/tests_ycbcr.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Defines unit tests for :mod:`colour.models.rgb.ycbcr` module.
+"""
+
+from __future__ import division, unicode_literals
+
+import numpy as np
+import unittest
+
+from colour.models.rgb.ycbcr import (rgb_to_YCbCr, YCbCr_to_rgb,
+                                     rgb_to_YcCbcCrc, YcCbcCrc_to_rgb,
+                                     RANGE, WEIGHT)
+
+__author__ = 'Colour Developers'
+__copyright__ = 'Copyright (C) 2013 - 2016 - Colour Developers'
+__license__ = 'New BSD License - http://opensource.org/licenses/BSD-3-Clause'
+__maintainer__ = 'Colour Developers'
+__email__ = 'colour-science@googlegroups.com'
+__status__ = 'Development'
+
+__all__ = ['Testrgb_to_YCbCr',
+           'TestYCbCr_to_rgb',
+           'Testrgb_to_YcCbcCrc',
+           'TestYcCbcCrc_to_rgb']
+
+
+class Testrgb_to_YCbCr(unittest.TestCase):
+    """
+    Defines :func:`colour.models.rgb.ycbcr.rgb_to_YCbCr` definition unit tests
+    methods.
+    """
+
+    def test_rgb_to_YCbCr(self):
+        """
+        Tests :func:`colour.models.rgb.ycbcr.rgb_to_YCbCr` definition.
+        """
+
+        np.testing.assert_almost_equal(
+            rgb_to_YCbCr(np.array([0.75, 0.75, 0.0])),
+            np.array([0.65842092, 0.17204301, 0.53060532]),
+            decimal=7)
+
+        np.testing.assert_almost_equal(
+            rgb_to_YCbCr(np.array([0.25, 0.5, 0.75]),
+                         K=WEIGHT['Rec.601'],
+                         outRange=RANGE['legal_10_YC_int']),
+            np.array([461, 662, 382]),
+            decimal=7)
+
+
+class TestYCbCr_to_rgb(unittest.TestCase):
+    """
+    Defines :func:`colour.models.rgb.ycbcr.YCbCr_to_rgb` definition unit tests
+    methods.
+    """
+
+    def test_YCbCr_to_rgb(self):
+        """
+        Tests :func:`colour.models.rgb.ycbcr.YCbCr_to_rgb` definition.
+        """
+
+        np.testing.assert_almost_equal(
```

Comment:
1. We try to at least do 3 unit tests but we will likely need more for those guys.
2. We will need to check the support for n-dimensional arrays and NAN too: https://github.com/colour-science/colour/blob/develop/colour/colorimetry/tests/tests_lightness.py#L58